libpod/container_internal.go

```diff
@@ -1119,7 +1119,7 @@ func (c *Container) cleanupRuntime(ctx context.Context) error {
 		return err
 	}
 
-	// If we were Stopped, we are now Exited, as we've removed ourself
+	// If we were Stopped, we are now Exited, as we've removed ourselves
```

Comment:
Minor preference for leaving as-is; I think it reads better as `ourself` since it's a single instance.

Reply:
Now fixed. I didn't even know that _ourself_ was a word. I learned something new today.

Reply:
One finds it necessary when referring to oneself using the royal "we"

Reply:
Aha,  interesting!